### ISSUE TITLE
Malwarebytes detected a "Malware.Heuristic.1001"

### ISSUE THREAD

**Issue by V0IIDx** [sotolko/csgo-market-sniper#9]
As soon as i tried to launch the script MalwareBytes intervened and blocked a "threat" : chromedriver.exe
in the location C:\Users\Asus\AppData\Local\Programs\Python\Python311\Lib\site-packages\chromedriver_autoinstaller\112\chromedriver.exe
Is it a false positive or a real virus, can you please reassure and convince me if it's not. 

ps: I'm from a thirdworld country, it's really tough out here and i'm desperate to make a quick buck from online sources such as steam market, it would really suck if my laptop and data get hijacked with viruses and spyware. 
![Capture](https://user-images.githubusercontent.com/130122892/230698383-36346a6f-28f9-4674-a6ec-b425c9316b45.PNG)

**Comment by sotolko** [comment]
I am hearing this for the first time. Malwarebytes is flagging [this](https://github.com/yeongbin-jo/python-chromedriver-autoinstaller), which is Python library that automatically installs chromedriver. And they have larger and a lot more popular repository than me. So I am pretty sure it is false positive.

**Comment by V0IIDx** [comment]
Yeah you're right, i guess it is a false positive sorry for the hassle :)
-Other thing :
I launched the script and everything was working fine for a while until steam page loaded this : "We were unable to service your request. Please try again later. E502 L3" which in turn made the script exit.
This error could be avoided by manually refreshing the page and the script carries on running.
Maybe you could include this error in the code so that the script auto refreshes the page when it encounters it ? 
 
![error](https://user-images.githubusercontent.com/130122892/230733957-f05f47e8-f4b7-4253-855a-79d104cc284f.png)
